mmv1/products/modelarmorglobal/Floorsetting.yaml

```
@@ -0,0 +1,261 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Floorsetting
+description: Description
+base_url: '{{parent}}/locations/{{location}}/floorSetting'
+update_mask: true
+id_format: '{{parent}}/locations/{{location}}/floorSetting'
+self_link: '{{parent}}/locations/{{location}}/floorSetting'
+create_url: '{{parent}}/locations/{{location}}/floorSetting'
+# This is a singleton resource that is already created, so create
+# is really an update, and therefore should be PATCHed.
+create_verb: 'PATCH'
+update_url: '{{parent}}/locations/{{location}}/floorSetting'
+update_verb: 'PATCH'
+# This is a singleton resource that cannot be deleted, so skip delete.
+exclude_delete: true
+custom_code:
+  custom_import: 'templates/terraform/custom_import/modelarmorglobal_floorsetting.go.tmpl'
+  post_create: 'templates/terraform/post_create/modelarmor_floorsetting_sleep.go.tmpl'
+  post_update: 'templates/terraform/post_create/modelarmor_floorsetting_sleep.go.tmpl'
+examples:
+    # Excluding tests as they modify a singleton resource and hence will result in race conditions
+  - name: 'modelarmor_floorsetting_basic'
+    primary_resource_id: 'floorsetting-basic'
+    test_vars_overrides:
+      parent: '"projects/ci-test-project-188019"'
+      location: '"global"'
+    exclude_test: true
+  - name: 'modelarmor_floorsetting_filter_config'
+    primary_resource_id: 'floorsetting-filter-config'
+    test_vars_overrides:
+      location: '"global"'
+      parent: '"projects/ci-test-project-188019"'
+      filter_config_rai_settings_rai_filters_0_filter_type: '"DANGEROUS"'
+      filter_config_rai_settings_rai_filters_0_confidence_level: '"MEDIUM_AND_ABOVE"'
+      sdp_settings_config_type: '"basic_config"'
+      filter_config_sdp_settings_basic_config_filter_enforcement: '"ENABLED"'
+      filter_config_pi_and_jailbreak_filter_settings_filter_enforcement: '"ENABLED"'
+      filter_config_pi_and_jailbreak_filter_settings_confidence_level: '"HIGH"'
+      filter_config_malicious_uri_filter_settings_filter_enforcement: '"ENABLED"'
+      enable_floor_setting_enforcement: true
```

```suggestion
    test_env_vars:
      project_id: 'PROJECT_NAME'
```

You can just specify the rest of variables in the configuration file